src/components/data-table/sort-filter-worker.ts

```diff
@@ -75,6 +70,10 @@ const sortData = (
       return -1;
     }
 
+    if (column.type !== "numeric") {
+      return sort * stringCompare(valA, valB, language);
```

Comment:
Should we check if the value is a string?

Reply:
We can use it if both values are string 👍